Products/CMFPlomino/PlominoForm.py

```diff
@@ -1335,6 +1335,37 @@ def validation_errors(self, REQUEST):
                     errors=json.dumps({'success': True}))
 
 
+    security.declarePrivate('_get_js_hidden_subforms')
+    def _get_js_hidden_subforms(self, REQUEST, doc, validation_mode=False):
+        hidden_forms = []
+        hidewhens = json.loads(
```

Comment:
Here you call a method which compute something, serialize it in JSON, and then you deserialize back.
We cannot do that. We must create a new method `getHidewhen` that would be used here and that would we used in `getHidewhenAsJSON` too.